lldb/test/API/lang/swift/async/stepping/step_out/TestSteppingOutAsync.py

```diff
@@ -0,0 +1,44 @@
+import lldb
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbtest as lldbtest
+import lldbsuite.test.lldbutil as lldbutil
+import re
+
+
+class TestCase(lldbtest.TestBase):
+
+    def check_and_get_frame_names(self, process):
+        frames = process.GetSelectedThread().frames
+        # Expected frames:
+        # ASYNC___0___ <- ASYNC___1___ <- ASYNC___2___ <- ASYNC___3___ <- Main
+        num_frames = 5
+        func_names = list(map(lambda frame: frame.GetFunctionName(), frames[:num_frames]))
```

Comment:
a more pythonic formation:

```suggestion
        func_names = [frame.GetFunctionName() for frame in frames[:num_frames]]
```

Reply:
Nice!